README.md

```diff
@@ -43,10 +43,8 @@ The following WDL features, that require more specific parsing or visualization,
 
 * [Imports](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#import-statements)
 * [Sub workflows](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#sub-workflows)
-* [Conditionals](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#conditionals)
-* [Scatter](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#scatter) - IN DEVELOPMENT
-* [Loop](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#loops) - IN DEVELOPMENT
-* [Workflow level resolution](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#workflow-level-resolution)
+* [Order when generating declarations of any step(workflow, call, if, scatter, while and task)](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#workflow-level-resolution)
+* [Declarations into the [call, if, scatter and while] blocks are not supported](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#conditionals)
```

Comment:
Thank you very much for this note! 

Would it be suitable to write:
**"Order of steps (workflow, call, if, scatter, while) in generated WDL code"** for the first case?